tests/Unit/NumericalAlgorithms/DiscontinuousGalerkin/Actions/Test_FluxCommunication.cpp

```diff
@@ -379,20 +403,19 @@ SPECTRE_TEST_CASE("Unit.DiscontinuousGalerkin.Actions.FluxCommunication",
   ActionTesting::set_phase(make_not_null(&runner),
                            metavariables::Phase::Testing);
 
-  runner.next_action<my_component>(self_id);
+  runner.template next_action<my_component>(self_id);
 
   // Here, we just check that messages are sent to the correct places.
   // We will check the received values on the central element later.
   {
-    CHECK(
-        runner
-            .nonempty_inboxes<my_component, fluxes_tag<flux_comm_types<2>>>() ==
-        std::unordered_set<ElementIndex<2>>{west_id, east_id, south_id});
+    CHECK(runner.template nonempty_inboxes<my_component,
+                                           fluxes_tag<flux_comm_types>>() ==
+          std::unordered_set<ElementIndex<2>>{west_id, east_id, south_id});
     const auto check_sent_data = [&runner, &self_id ](
         const ElementId<2>& id, const Direction<2>& direction) noexcept {
-      const auto& inboxes = runner.inboxes<my_component>();
+      const auto& inboxes = runner.template inboxes<my_component>();
```

Comment:
[optional] I think there is also non-member for inboxes (and many below). Again, I'll let you decide whether you want to do that or not